app/assets/javascripts/select-kit/addon/components/bulk-select-bookmarks-dropdown.js

```diff
@@ -0,0 +1,111 @@
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+import Bookmark from "discourse/models/bookmark";
+import i18n from "discourse-common/helpers/i18n";
+import DropdownSelectBoxComponent from "select-kit/components/dropdown-select-box";
+
+const _customButtons = [];
+const _customActions = {};
+
+export function addBulkDropdownAction(name, customAction) {
+  _customActions[name] = customAction;
+}
+
+export function addBulkDropdownButton(opts) {
```

Comment:
I don't think we need this for bookmarks. In the topics list, this is linked to a plugin API endpoint that lets plugins add their own buttons and actions.